wolf-data/src/Wolf/Data/Export/Types.hs

```diff
@@ -89,3 +105,48 @@ instance ToJSON Repo where
             , "notes" .= repoNotes
             , "suggestions" .= repoSuggestions
             ]
+
+type Warn = [Problem]
+
+data Problem
+    = WarnMissingNoteIndex PersonUuid
+    | WarnMissingNote NoteUuid
+    | WarnMissingRelevantPerson PersonUuid
+                                NoteUuid
+    | WarnMissingRelevantNote NoteUuid
+                              PersonUuid
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Problem
+
+instance FromJSON Problem
+
+data Err =
+    NoInitFile
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Err
+
+instance FromJSON Err
+
+prettyPrintErr :: Err -> String
+prettyPrintErr NoInitFile =
+    "Error: the wolf repository is not or poorly initialised."
+
+prettyPrintProblem :: Problem -> String
+prettyPrintProblem (WarnMissingNoteIndex pu) =
```

Comment:
Why is this a warning? If a note doesn't have any notes, it doesn't need a note index.

Reply:
If you would look at where it's used, you'd know it's not a warning I throw because just if a noteindex doesn't. Instead, I look up notes, look at the relevant people corresponding to these notes. Those people should have a noteindex, and the noteindex should contain the aforementioned note.
I'll add the NoteUuid to the warning, if that makes it clearer.